sdk_container/src/third_party/coreos-overlay/coreos-base/misc-files/misc-files-0.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2023 The Flatcar Maintainers.
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION='Flatcar bashrc files'
+HOMEPAGE='https://www.flatcar.org/'
+
+LICENSE='Apache-2.0'
+SLOT='0'
+KEYWORDS='amd64 arm64'
+
+# No source directory.
+S="${WORKDIR}"
+
+# Versions listed below are version of packages that shedded the
+# modifications in their ebuilds.
+RDEPEND="
+	>=app-shells/bash-5.2_p15-r2
+"
+
+# Use absolute paths to be clear about what locations are used. The
+# dosym below will make relative paths out of them.
+#
+# NOTE: The symlinks under /usr that point to entries under /etc will
+# be retargeted to /usr/share/flatcar/etc when building an image
+# later. Do not make them point to /usr/share/flatcar/etc here,
+# because this directory does not exist now and won't exist at all
+# during the whole packages build phase. We don't want to create
+# dangling symlinks because the packages build phase also runs a root
+# check at the end, and it will bail out on dangling symlinks.
+COMPAT_SYMLINKS=(
```

Comment:
Not sure if emerge supports bash 4 features, if yes it could use https://stackoverflow.com/a/3467959

Reply:
[EAPI 8 supports bash 5](https://devmanual.gentoo.org/ebuild-writing/eapi/index.html#eapi-8-ebuild-format), so yes, we could use a hashmap. Good idea, I'll switch to it.